src/commands/deploy/promptToSaveDeployDefaults.ts

```diff
@@ -15,7 +15,7 @@ export async function promptToSaveDeployDefaults(context: IActionContext, node:
     const defaultWebAppToDeploySetting: string | undefined = getWorkspaceSetting(constants.configurationSettings.defaultWebAppToDeploy, workspacePath);
     // only prompt if setting is unset
     if (!defaultWebAppToDeploySetting) {
-        const saveDeploymentConfig: string = localize('showDeploymentConfig', 'Always deploy the workspace "{0}" to "{1}"?', path.basename(workspacePath), node.root.client.fullName);
+        const saveDeploymentConfig: string = localize('showDeploymentConfig', 'Always deploy the workspace "{0}" to "{1}"?', path.relative(workspacePath, deployPath) || path.basename(workspacePath), node.root.client.fullName);
```

Comment:
Would "workspace" still work here? or should we use something more like "directory/folder"?

Reply:
I think workspace is probably still fine wordage, but did we want it to only show the subfolder name?  I'm not sure how I feel about it, but it seems like having the full path could be more useful, but I understand the concern if the subfolder was nested pretty deeply.

Reply:
I agree, full path does give more info especially for nested folders. 

But I think it makes sense to fix this wording issue as in the case of deploying a nested folder, it get's confusing to display the parent folder for the prompt. 
Also, the `saveDeployDefaults()` behavior doesn't match the wording, this fix will match the wording with the function's behavior.